docs/content/ssr.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Server-side rendering with Primer React
```

Comment:
Thank you for writing docs! 💖

Reply:
Can you add this doc to the side nav? https://github.com/primer/react/blob/main/docs/src/%40primer/gatsby-theme-doctocat/nav.yml